packages/tenant-process/src/services/tenantService.ts

```diff
@@ -159,7 +225,7 @@ export function tenantServiceBuilder(
         };
 
         logger.info(
-          `Creating tenant with external id ${tenantSeed.externalId} via SelfCare request"`
+          `Updating tenant with external id ${tenantSeed.externalId} via SelfCare request"`
```

Comment:
I modified the log, because at this point there is the updating of the tenant, not the creation

Reply:
> I modified the log, because at this point there is the updating of the tenant, not the creation

In this case move the log after the condition, since there could be either a `create` or an `update`based on that